src/App.test.js

```diff
@@ -1,13 +1,12 @@
-import React from 'react'
-import { render } from '@testing-library/react'
-import { shallow, mount } from 'enzyme'
```

Comment:
Do we want this here?

Reply:
The only reason I can think to keep it, is to have is as a placeholder for an applicant that do its test with enzyme.